style.css

```diff
@@ -422,6 +501,238 @@ form textarea:focus {
   padding-left: 22%;
 }
 
+.primary,
 .social-media-icons {
   padding-top: 3em;
 }
+
+div {
+  display: block;
+}
+
+#section-contact p {
+  font-family: "inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  margin-top: 1em;
+  margin-bottom: 3em;
+}
+
+#textarea {
+  background-image: url(./imges/Icon\ -\ Resizer\ Handle.svg);
+  background-repeat: no-repeat;
+  background-position: left 325px, 200px 385px, right 280px;
+  background-size: 130px;
+}
+
+#mobile-menu {
+  height: 85vh;
+  border-bottom-right-radius: 2em;
+  border-bottom-left-radius: 2em;
+  margin-bottom: 2em;
+  mix-blend-mode: normal;
+  box-shadow: 0 1em 1em rgba(37, 47, 137, 0.08);
+}
+
+#mobile-menu > ul {
+  clear: right;
+  padding-left: 0;
+}
+
+.menu-link {
+  display: block;
+  text-decoration: none;
+  margin: 1em 1em;
+  color: #3a4a42;
+  font-size: x-large;
+  font-weight: 700;
+}
+
+.hidden {
+  display: none;
+}
+
+.social-media-icons li:hover {
+  transform: scale(1.3);
+}
+
+@media (min-width: 768px) {
+  /* Standard CSS rules */
+  html {
+    box-sizing: border-box;
+  }
+
+  header > a,
+  header > img {
+    display: none;
+  }
+
+  header {
+    padding: 0;
+  }
+
+  #headline {
+    background-image: url("./imges/Desktop/Header-llustration.png");
+    background-size: 106% 16%;
+  }
```

Comment:
- [ ] Good job so far, I think it would be better if you arrange your background image 🙃 
- [ ] Also you should make sure your text has the same font as the text in the Figma design 😉

Your design

<img width="778" alt="Screen Shot 2022-11-11 at 23 44 01" src="https://user-images.githubusercontent.com/74501523/201436766-40bf6684-df52-4d1b-b013-e35631cd0abc.png">

Figma design

<img width="723" alt="Screen Shot 2022-11-11 at 23 45 55" src="https://user-images.githubusercontent.com/74501523/201436799-6f272703-cce1-43b0-9e98-56a30726d770.png">

Reply:
- [ ] I think it's better to use the same font size and weight as supplied on the Figma template. Stick to the template as much as possible.

Figma template

<img width="1000" alt="Screen Shot 2022-11-12 at 8 03 36 PM" src="https://user-images.githubusercontent.com/47673311/201498025-a74e670c-08a5-492a-8ebc-a27eac41c0de.png">